tests/test_username_api.py

```diff
@@ -134,6 +129,6 @@ def test_format_checking(self, website, username):
         json_resp = json.loads(resp.get_data().decode())
         url = username_api.get_profile_url(website, username)
         assert {
-           'possible': False,
-           'url': url
+            'possible': False,
```

Comment:
I'm curious that coala doesn't fix this indentation/

Reply:
by the way should I remove this change?

Reply:
Either that or move it to a new commit

Reply:
sure

Reply:
@andrewda done